### DISCUSSION TITLE
GitHub Copilot Access - README

### DISCUSSION THREAD

**Discussion by fraxinus1098** [yale-research-job-board/.github#25] [Announcements]
If you're trying to get access to GitHub Copilot, please request it here:
1. https://education.github.com/discount_requests/application

See these for trouble shooting
1. https://docs.github.com/en/copilot/managing-copilot/managing-copilot-as-an-individual-subscriber/managing-your-github-copilot-pro-subscription/getting-free-access-to-copilot-pro-as-a-student-teacher-or-maintainer
2. https://github.com/orgs/community/discussions/137101

As far as I understand (someone correct me if I'm wrong):
1. Easiest: You have a GitHub account under your personal email, and you can request the student access to Copilot following the normal link request and verifying your student email
2. You have a GitHub account under your yale.edu email, and you should be able to go through the same process? I think that there are some hiccups along the way, so see the trouble shooting links and post your questions here and someone else can help.

**Comment by fraxinus1098** [comment]
I know that a few of you requested access to Copilot via the org, but please do it through the link above instead.

**Comment by kljensen** [comment]
> If you're trying to get access to GitHub Copilot,
> 
> 1. Easiest: You have a GitHub account under your personal email, and you can request the student access to Copilot following the normal link request and verifying your student email
> 
> 2. You have a GitHub account under your yale.edu email, and you should be able to go through the same process? I think that there are some hiccups along the way, so see the trouble shooting links and post your questions here and someone else can help. 

Yes. "Student developer pack". Free copilot
